static/app/views/settings/organizationAuthTokens/authTokenRow.tsx

```diff
@@ -161,6 +168,7 @@ const Actions = styled('div')`
 const LastUsedDate = styled('div')`
   display: flex;
   align-items: center;
+  gap: 0.25em;
```

Comment:
I understand why you added 0.25em here but we usually use the values we have available in the `space`

Reply:
ah, true - will adjust this to use space 👍